installer/conf/omsagent.d/patch_management.conf

```diff
@@ -23,4 +23,29 @@
 <filter oms.patch_management_immediate_run>
   type filter_patch_management
   log_level warn
+</filter>
+
+<source>
+    @type tail
+    tag oms.update_progress
+    path /var/log/apt/history.log
+    pos_file /var/opt/microsoft/omsagent/run/history.log.pos
+    keep_time_key true
+    format multiline
+    format_firstline /Start-Date:/
+    format1 /Start-Date: (?<start-date>.*)\n/
+    format2 /Commandline: (?<apt-logs>.*\n)/
+    read_from_head true
+    log_level trace
+</source>
+
+<filter oms.update_progress>
+    @type grep
+    regexp1 apt-logs Requested
```

Comment:
The history file on my Ubuntu 14.04 looks like this:

Start-Date: 2016-11-03  04:00:29
Commandline: apt-get install samba
Install: update-inetd:amd64 (4.43, automatic), libaio1:amd64 (0.3.109-4, automatic), python-dnspython:amd64 (1.11.1-1build1, automatic), tdb-tools:amd64 (1.3.8-0ubuntu0.14.04.1, automatic), samba:amd64 (4.3.11+dfsg-0ubuntu0.14.04.1), samba-dsdb-modules:amd64 (4.3.11+dfsg-0ubuntu0.14.04.1, automatic), libfile-copy-recursive-perl:amd64 (0.38-1, automatic), samba-vfs-modules:amd64 (4.3.11+dfsg-0ubuntu0.14.04.1, automatic), attr:amd64 (2.4.47-1ubuntu1, automatic)
End-Date: 2016-11-03  04:00:46

No "Requested" in the message when I install a new package.